pages/index.js

```diff
@@ -1,3 +1,27 @@
+import styled from "styled-components";
+import FoodCard from "../components/FoodCard";
+import useSWR from "swr";
+import { fetcher } from "../helpers/api";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  const { data: data, error } = useSWR("/api/recipes", fetcher);
```

Comment:
I would recommend to rename data to make it more descriptive. If you do so, do not forget to change it for this whole file 🙂 

```suggestion
  const { data: recipes, error } = useSWR("/api/recipes", fetcher);
```